ListKit.xcodeproj/project.pbxproj

```diff
@@ -389,21 +389,24 @@
 			attributes = {
 				LastSwiftMigration = 0700;
 				LastSwiftUpdateCheck = 0700;
-				LastUpgradeCheck = 0700;
+				LastUpgradeCheck = 0800;
```

Comment:
ah sorry, I didn't realize that! will submit a new PR shortly, thanks a lot for the hint!